aiida_cp2k/calculations/__init__.py

```diff
@@ -30,16 +30,17 @@ class Cp2kCalculation(CalcJob):
     _DEFAULT_OUTPUT_FILE = 'aiida.out'
     _DEFAULT_PROJECT_NAME = 'aiida'
     _DEFAULT_RESTART_FILE_NAME = _DEFAULT_PROJECT_NAME + '-1.restart'
+    _DEFAULT_TRAJECT_FILE_NAME = _DEFAULT_PROJECT_NAME + '-pos-1.dcd'
     _DEFAULT_PARENT_CALC_FLDR_NAME = 'parent_calc/'
     _DEFAULT_COORDS_FILE_NAME = 'aiida.coords.xyz'
-    _DEFAULT_PARSER = 'cp2k'
+    _DEFAULT_PARSER = 'cp2k_base_parser'
 
     @classmethod
     def define(cls, spec):
         super(Cp2kCalculation, cls).define(spec)
 
         # Input parameters
-        spec.input('parameters', valid_type=Dict, help='the input parameters')
+        spec.input('parameters', valid_type=Dict, required=False, help='the input parameters')
```

Comment:
This is because you can expose all the inputs from the CP2K calculation in a workchain that creates the input parameters internally, but you still have the option of forcing certain parameters of the calculation. One could exclude this input parameters in the workchain and create another input that is required=False, but it is a more elaborate option.

Reply:
Hmm, I think it would would work anyways as long as you don't expose the input `cp2k.parameters` in the multistage work chain. And, as far as I can see, you do not expose it.

Reply:
See the later comment on `expose input`

Reply:
Agree on  not changing it